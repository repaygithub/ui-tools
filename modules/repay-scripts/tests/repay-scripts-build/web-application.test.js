const TestSetup = require('../test-setup')

beforeEach(() => jest.setTimeout(1000 * 60 * 5))

describe('@repay/repay-scripts', () => {
  it('should build a web application', async () => {
    const test = new TestSetup('web-application')
    await test.setup()
    await test.exec(`yarn repay-scripts build --babel-env test index.js`)

    test.startServer()
    const page = await test.getPage()
    await page.goto('http://localhost:8100')
    const content = await page.$eval('.app-root', el => el.outerHTML)
    expect(content).toMatchSnapshot()
  })
})
