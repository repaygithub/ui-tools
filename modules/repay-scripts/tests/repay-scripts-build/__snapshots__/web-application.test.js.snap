// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@repay/repay-scripts should build a web application 1`] = `"<div class=\\"app-root\\">This is the application root</div>"`;
